content/swiftui/concepts/views/terms/scrollview/scrollview.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: 'ScrollView'
+Description: 'Scrollable view within the scrollable content region.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'ScrollView'
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`ScrollView`** display the content within the scrollable content region. Based on platform appropriate scroll gestures, or how the code works, the view can scroll vertically, horizontally or both.
+
+## Syntax
+
+```pseudo
+var body: some View {
+    ScrollView(.vertical) {
+        Subviews here
+    }
+}
+```
+
+```pseudo
+var body: some View {
+    ScrollView(.horizontal) {
+        Subviews here
+    }
+}
+```
```

Comment:
Can we combine these? Instead of the specific `.vertical`/`.horizontal`, let's use "modifiers here"

Reply:
Will do!